Makefile

```diff
@@ -5,15 +5,18 @@ all: default
 VERSION="0.0" # dummy for now
 GIT_COMMIT=$(shell git rev-list -1 HEAD)
 
-GO_COMPILE=linuxkit/go-compile:4513068d9a7e919e4ec42e2d7ee879ff5b95b7f5@sha256:bdfadbe3e4ec699ca45b67453662321ec270f2d1a1dbdbf09625776d3ebd68c5
+GO_COMPILE=mor1/go-compile:021f2dca9228cdd27cffc304aafae1b1f900bfc2
 
 MOBY?=bin/moby
 LINUXKIT?=bin/linuxkit
 GOOS=$(shell uname -s | tr '[:upper:]' '[:lower:]')
-GOARCH=amd64
+GOARCH=arm64
```

Comment:
you should not be setting this unconditionally, you should use `uname` to find it